providers/implementations/ciphers/cipher_aes_gcm_hw_ppc.inc

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright 2001-2021 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2021- IBM Inc. All rights reserved
```

Comment:
I do not think we accept such additional copyright notices anymore in the code.

Reply:
got it.  Will fix.